packages/util-dev/src/TestLogger.ts

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable prettier/prettier */
+/* eslint key-spacing: ["error", { align: { afterColon: true, beforeColon: false, on: "value" } }] */
+import * as envalid from 'envalid';
+import { InspectOptions, inspect } from 'util';
+import { Writable } from 'stream';
+import JSONbig from 'json-bigint';
+
+const logLevels = { debug: 2, error: 5, fatal: 6, info: 3, trace: 1, warn: 4 };
+const logLevelLabels = ['', 'TRACE  ', 'DEBUG  ', 'INFO   ', 'WARNING', 'ERROR  ', 'FATAL  '];
+
+type LogLevel = keyof typeof logLevels;
+type LoggerEntry = [LogLevel, number];
+
+/**
+ * The base log function
+ */
+export type LogFunction = (...args: unknown[]) => void;
+
+/**
+ * Recorder messages is an Array of Objects of this type
+ */
+export type LoggedMessage = { level: LogLevel; message: unknown[] };
+
+/**
+ * The base logger object
+ */
+export type Logger = { [l in LogLevel]: LogFunction };
+
+/**
+ * A unit test dedicated logger. Check
+ * https://github.com/input-output-hk/cardano-js-sdk/tree/master/packages/util-dev#testlogger for details.
+ */
+export type TestLogger = Logger & { messages: LoggedMessage[] };
+
+type TestStream = Writable & { columns?: number; isTTY?: boolean };
+
+/**
+ * Options for createLogger
+ */
+export interface TestLoggerOptions {
+  /**
+   * The environment variables to use. Default: process.env - Used to test TestLogger
+   */
+  env?: NodeJS.ProcessEnv;
+
+  /**
+   * If true, the logger will records all the logged values
```

Comment:
Is the default 'false'?

Reply:
b6849e8a6 should make the question disappear.